CHANGELOG.md

```diff
@@ -4,6 +4,63 @@ This document includes a curated changelog for each release. We also publish a c
 a [GitHub release](https://github.com/nginxinc/nginx-gateway-fabric/releases), which, by contrast, is auto-generated
 and includes links to all PRs that went into the release.
 
+## Release 1.3.0
```

Comment:
Looks like there are a few release notes missing

Reply:
Actually might be just one: 
- GRPCRoute support: https://github.com/nginxinc/nginx-gateway-fabric/pull/1835

Reply:
https://github.com/nginxinc/nginx-gateway-fabric/pull/2047

Reply:
https://github.com/nginxinc/nginx-gateway-fabric/pull/1829